Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/DocumentPropertyEditor.cpp

```diff
@@ -545,6 +608,193 @@ namespace AzToolsFramework
+        auto sharePrior = AZ::Dpe::Nodes::PropertyEditor::SharePriorColumn.ExtractFromDomNode(domArray).value_or(false);
+        auto minimumWidth = AZ::Dpe::Nodes::PropertyEditor::UseMinimumWidth.ExtractFromDomNode(domArray).value_or(false);
+        auto descriptionString = AZ::Dpe::Nodes::PropertyEditor::Description.ExtractFromDomNode(domArray).value_or("");
+        auto shouldDisable = AZ::Dpe::Nodes::PropertyEditor::Disabled.ExtractFromDomNode(domArray).value_or(false);
+
+        // Check for a widget in the previous column
+        int priorColumnIndex = -1;
+        for (int searchIndex = static_cast<int>(domIndex) - 1; (priorColumnIndex == -1 && searchIndex >= 0); --searchIndex)
+        {
+            priorColumnIndex = m_columnLayout->indexOf(m_domOrderedChildren[searchIndex]);
+        }
+
+        auto foundEntry = m_domOrderToAttributeInfo.find(domIndex);
+
+        // If there is no existing attribute entry in the map, try to create a new one
+        if (foundEntry == m_domOrderToAttributeInfo.end())
+        {
+            bool validAttribute = false;
+            AttributeInfo* newAttribute = new AttributeInfo;
+
+            // Check Alignment
+            if (alignment != AZ::Dpe::Nodes::PropertyEditor::Align::UseDefaultAlignment)
+            {
+                newAttribute->m_alignment = alignment;
+                validAttribute = true;
+            }
+
+            // Check SharePrior
+            if (sharePrior)
+            {
+                AZ_Assert(priorColumnIndex != -1, "Tried to share column with an out of bounds index!");
+                if (priorColumnIndex != -1)
+                {
+                    m_columnLayout->AddSharePriorColumn(priorColumnIndex, domIndex);
+                    newAttribute->m_sharePriorColumn = true;
+                    validAttribute = true;
+                }
+            }
+            // Check MinimumWidth
+            if (minimumWidth)
+            {
+                newAttribute->m_minimumWidth = true;
+                validAttribute = true;
+            }
+
+            //Check DescriptionString
+            if (!descriptionString.empty())
+            {
+                if (toolTip().isEmpty())
+                {
+                    setToolTip(QString::fromUtf8(descriptionString.data(), aznumeric_cast<int>(descriptionString.size())));
+                }
+                if (childWidget->toolTip().isEmpty())
+                {
+                    setToolTip(QString::fromUtf8(descriptionString.data(), aznumeric_cast<int>(descriptionString.size())));
+                }
+                newAttribute->m_descriptionString = descriptionString;
+                validAttribute = true;
+            }
+
+            // Check Disabled
+            if (shouldDisable)
+            {
+                childWidget->setEnabled(false);
+                validAttribute = true;
+            }
```

Comment:
Here too, or maybe just here? One or both of these needs an else case for re-enabling a widget when the Disabled attribute is cleared.